company/careers.md

```diff
@@ -10,7 +10,7 @@ We're hiring! Check out our open positions:
 
 - [Director of Engineering - Platform and Infrastructure](https://jobs.lever.co/sourcegraph/8fcbaade-e511-4d93-bd9b-9cc7ec3438af)
 - [Engineering Manager - Cloud Application and Core Services](../handbook/engineering/hiring/engineering-manager-cloud.md)
-- [Software Engineer - Frontend](../handbook/engineering/hiring/software-engineer-frontend.md)
```

Comment:
@jeanduplessis is there any reason to not delete the other file?

Reply:
There are blog posts linking to this, so linting fails because of unresolvable references. What's best practice here? Should I just unlink the blog post?